README.md

```diff
@@ -1,18 +1,22 @@
-# Data Model Library
-[![Build Status](https://travis-ci.org/qbicsoftware/data-model-lib.svg?branch=master)](https://travis-ci.com/qbicsoftware/data-model-lib)
-[![Code Coverage]( https://codecov.io/gh/qbicsoftware/data-model-lib/branch/master/graph/badge.svg)](https://codecov.io/gh/qbicsoftware/data-model-lib)
-[![Latest Release ](https://img.shields.io/github/v/release/qbicsoftware/data-model-lib.svg)](https://github.com/qbicsoftware/data-model-lib/releases)
-![Java Language](https://img.shields.io/badge/language-java-blue.svg)
-![Groovy Language](https://img.shields.io/badge/language-groovy-blue.svg)
+<div align="center">
 
-Data Model Library - A collection of QBiC data models.
+#Data Model Library
+
+[![Build Maven Package](https://github.com/qbicsoftware/data-model-lib/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/data-model-lib/actions/workflows/build_package.yml)
+[![Run Maven Tests](https://github.com/qbicsoftware/data-model-lib/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/data-model-lib/actions/workflows/run_tests.yml)
+[![CodeQL](https://github.com/qbicsoftware/data-model-lib/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/data-model-lib/actions/workflows/codeql-analysis.yml)
+[![release](https://img.shields.io/github/v/release/qbicsoftware/data-model-lib?include_prereleases)](https://github.com/qbicsoftware/data-model-lib/releases)
 
-## Author
-Created by Andreas Friedrich, Luis de la Garza, Sven Fillinger.
+[![license](https://img.shields.io/github/license/qbicsoftware/data-model-lib)](https://github.com/qbicsoftware/data-model-lib/blob/main/LICENSE)
+![language](https://img.shields.io/badge/language-groovy,%20java-blue.svg)
+
+Data Model Library - A collection of QBiC data models.
+</div>
 
-Overview:
+##Overview:
```

Comment:
Missing Space leads to showing the markdown code instead of the formatted result
```suggestion
## Overview:
```